mx.truffleruby/mx_truffleruby.py

```diff
@@ -19,7 +19,7 @@
     import mx_truffleruby_benchmark
 
 _suite = mx.suite('truffleruby')
-rubyDists = ['TRUFFLERUBY', 'TRUFFLERUBY-TEST']
+root = _suite.dir
```

Comment:
Does it work if a parent project is importing both projects anyway? I'll try it after we've merged this.